include/small/rlist.h

```diff
@@ -76,9 +78,17 @@ rlist_add(struct rlist *head, struct rlist *item)
 	item->next->prev = item;
 }
 
+__attribute__((no_sanitize_address))
+inline static void
+rlist_add_no_asan(struct rlist *head, struct rlist *item)
```

Comment:
As discussed f2f - added `always_inline` attribute where required so that it is possible to disable ASAN instrumentation if caller has `no_sanitize_address` attribute.

Reply:
OK. Let's add the `always_inline` attribute (in a separate commit or even PR). The rlist methods should always be inlined anyway.

Reply:
Done.